src/ViewerComponents/ResourceViewer.svelte

```diff
@@ -0,0 +1,415 @@
+<script>
+    import {onMount} from 'svelte';
+    import {token, lists, ontologies} from '../store';
+
+    export let resource, ontology, server, user, shortname, shortcode;
+    let properties = {}
+    let error = false;
+
+    onMount(async () => {
+        // For later
+        // await login();
+    })
+
+    /**
+     * Logs in to the provided server and saves the token into the store.
+     *
+     * @returns {Promise<void>}
+     */
+    async function login() {
```

Comment:
Same as above, suggest to move this to dsp services and move the request somewhere else

Reply:
Will happen later